composer.json

```diff
@@ -80,8 +81,7 @@
         "sonata-project/formatter-bundle": "^3.4",
         "sonata-project/notification-bundle": "^3.3",
         "sonata-project/seo-bundle": "^2.5",
-        "symfony/phpunit-bridge": "^4.0",
-        "symfony/security-csrf": "^2.8 || ^3.2 || ^4.0"
```

Comment:
Why are you removing this?

Reply:
Because only `symfony/security-bundle` should (have been?) added in #1398, and security-csrf and security-core are part of security-bundle

Reply:
Ok I'll let @jordisala1991 do the review then :+1:

Reply:
https://github.com/sonata-project/SonataMediaBundle/blob/100ecd16c8a5eae3dc8183949f0b88a325cbd40e/tests/Controller/GalleryAdminControllerTest.php#L35